src/SignalR/clients/ts/FunctionalTests/yarn.lock

```diff
@@ -3550,20 +3550,10 @@ wrappy@1:
   dependencies:
     async-limiter "~1.0.0"
 
-ws@^7.2.3:
-  version "7.5.5"
-  resolved "https://registry.yarnpkg.com/ws/-/ws-7.5.5.tgz#8b4bc4af518cfabd0473ae4f99144287b33eb881"
-  integrity sha512-BAkMFcAzl8as1G/hArkxOxq3G7pjUqQ3gzYbLL0/5zNkph70e+lCoxBGnm6AW1+/aiNeV4fnKqZ8m4GZewmH2w==
-
-ws@^7.4.5:
-  version "7.4.5"
-  resolved "https://registry.yarnpkg.com/ws/-/ws-7.4.5.tgz#a484dd851e9beb6fdb420027e3885e8ce48986c1"
-  integrity sha512-xzyu3hFvomRfXKH8vOFMU3OguG6oOvhXMo3xsGy3xWExqaM2dxBbVxuD99O7m3ZUFMvvscsZDqxfgMaRr/Nr1g==
-
-ws@~8.2.3:
-  version "8.2.3"
-  resolved "https://registry.yarnpkg.com/ws/-/ws-8.2.3.tgz#63a56456db1b04367d0b721a0b80cae6d8becbba"
-  integrity sha512-wBuoj1BDpC6ZQ1B7DWQBYVLphPWkm8i9Y0/3YdHjHKHiohOJ1ws+3OccDWtH+PoC9DZD5WOTrJvNbWvjS6JWaA==
+ws@>=7.4.6, ws@^7.2.3, ws@^7.4.5, ws@~8.2.3:
+  version "8.6.0"
```

Comment:
I _guess_ this is better but now the 7.x.y dependencies are getting updated across a major version boundary. @BrennanConroy is JS testing enough to catch any problems that may cause❔

Reply:
This version is only used for a single test AFAIK and it's sort of a silly test IMO.

The majority of our tests that use WS actually get WS from https://github.com/dotnet/aspnetcore/tree/main/src/SignalR/clients/ts/signalr which is using 7.4.6